<launch>
    <!-- Load parameters from YAML -->
    <arg name="config_file" default="$(find icetrack_mapping)/config/general.yaml" />
    <rosparam file="$(arg config_file)" command="load"/>

    <rosparam file="$(find icetrack_mapping)/config/icetrack_mapping.yaml" command="load"/>

    <!-- Rosbag node requres sim time -->
    <param name="use_sim_time" value="true" />

    <!-- Launch node -->
    <node 
        name="rosbag_node" pkg="icetrack_mapping" type="rosbag_mapping_node" output="screen" required="true">
    </node>
</launch>